ui/src/components/footer/ChurchAppDownloadButton.js

```diff
@@ -0,0 +1,28 @@
+import { Box, VStack, HStack, Image, Text, Link } from "@chakra-ui/react";
+export function ChurchAppDownloadButton() {
+  return (
+    <VStack minW="50%">
+      <Box height="25px">
+        <Text color="white" fontSize="md">
+          Get our Church App
+        </Text>
+      </Box>
+      <HStack>
+        <Link href="https://tithely.app.link/harvest-mission-community-church-hong-kong">
```

Comment:
Outlinks should use a `target: "_blank"` attribute. This is done so when it redirects someone out of our site, it will open the link in a new tab instead of leaving our site

Reply:
added target ='_blank'